tillerinobot/src/main/java/tillerino/tillerinobot/lang/Farsi.java

```diff
@@ -0,0 +1,217 @@
+	 * @return
+	 */
+	public String unresolvableName(String exceptionMarker, String name) {
+		return "Esmet waasam mashkooke. Banned shodi? Age intori nist, sari begoo behem: @Tillerino ya /u/Tillerino (reference "
+				+ exceptionMarker + ")";
+	}
+
+	@Override
+	public String excuseForError() {
+		return "Waiiiiiiiiiiiii~ hawaasam part shod! Bebaghshid, lotfan ye baare dige, chi gofti?";
+	}
+
+	@Override
+	public String complaint() {
+		return "Damet garm, ferestaadam. Tillerino negaa mikone ke to dobaare chiro kharaab kardi ;)";
+	}
+
+	@Override
+	public void hug(final IRCBotUser user, OsuApiUser apiUser) {
+		user.message("Bia injaa binam!");
+		user.action("ra " + apiUser.getUserName() + " baghal mikone");
+	}
+
+	@Override
+	public String help() {
+		return "Bedrood~ Man aankesiam ke Tillerino-ro kosht wa account-esho dozdid. Shookhi kardam, wali man hanooz account-amo estefaade mikonam."
+				+ " https://twitter.com/Tillerinobot ro check kon waase status-o updates!"
+				+ " https://github.com/Tillerino/Tillerinobot/wiki ro bebin waase command-ha!";
+	}
+
+	@Override
+	public String faq() {
+		return "https://github.com/Tillerino/Tillerinobot/wiki/FAQ ra bebin waaseye FAQ!";
+	}
+	
+	@Override
+	public String featureRankRestricted(String feature, int minRank, OsuApiUser user) {
+		return "Bebaghshid, wali aalan " + feature + " felan player-ha mitoonan estefaade konan ke rank-eshoon baalaatar az " + minRank + " hast.";
+	}
+	
+	@Override
+	public String mixedNomodAndMods() {
+		return "Hawaaseto jam kon, nomod ba mods yani chi~?";
+	}
+	
+	@Override
+	public String outOfRecommendations() {
+		return "Man dige fekram nemirese, harchi toonestam recommend kardam."
+				+ " Age mikhaai, option-haye digei estefaade kon, yaa type kon !reset. Nemidooni chekaar koni, daad bezan !help";
+	}
+
+	@Override
+	public String notRanked() {
+		return "In beatmap ranked ke nist!";
+	}
+
+	@Override
+	public void optionalCommentOnNP(IRCBotUser user,
+			OsuApiUser apiUser, BeatmapMeta meta) {
+		// regular Tillerino doesn't comment on this
+	}
+
+	@Override
+	public void optionalCommentOnWith(IRCBotUser user, OsuApiUser apiUser,
+			BeatmapMeta meta) {
+		// regular Tillerino doesn't comment on this
+	}
+
+	@Override
+	public void optionalCommentOnRecommendation(IRCBotUser user,
+			OsuApiUser apiUser, Recommendation meta) {
+		// regular Tillerino doesn't comment on this
+	}
+	
+	@Override
+	public boolean isChanged() {
+		return false;
+	}
+
+	@Override
+	public void setChanged(boolean changed) {
+		
+	}
+
+	@Override
+	public String invalidAccuracy(String acc) {
+		return "Accuracy-ro eshtebaa type nakardi? \"" + acc + "\" yani chi?";
+	}
+
+	@Override
+	public String noPercentageEstimates() {
+		return "Bebaghshid, wali endafe nemitoonam begam in chie.";
+	}
+
+	@Override
+	public void optionalCommentOnLanguage(IRCBotUser user, OsuApiUser apiUser) {
+		user.message("ikhebepicmuis-jan learned me to talk Farsi!"); 
```

Comment:
I don't know if I was unclear or if you have a different opinion on this. I think this line should be in Farsi. Don't you think?